src/main/java/net/glowstone/command/minecraft/FunctionCommand.java

```diff
@@ -37,13 +37,7 @@ public boolean execute(CommandSender sender, String label, String[] args) {
             return false;
         }
         String functionName = args[0];
-        CommandFunction function = null;
-        for (CommandFunction f : world.getFunctions()) {
-            if (f.getFullName().equals(functionName)) {
-                function = f;
-                break;
-            }
-        }
+        CommandFunction function = world.getFunctions().get(functionName);
         if (function == null) {
```

Comment:
Use `getFunctions().containsKey()` check here, and `get` the function after that check.